### DISCUSSION TITLE
New Plugin: Copy Text

### DISCUSSION THREAD

**Discussion by gpsblues** [GetPublii/Publii#1788] [Useful Resources]
Hello Community!

I’m continuing my experiments with creating Publii plugins, and I’m released **[Copy Text](https://github.com/gpsblues/Publii-Plugin-Copy-text)**.

This plugin makes it easy to add a copy-to-clipboard function to any HTML element on your site.

**How It Works**
To activate the copy function, simply add a custom class to the HTML tag where you want to enable copying (default is copy-text). Once applied, the element will show an icon that lets visitors copy the content with one click.

**Key Features:**
Easy Setup: Just add a custom class to enable copying.
Basic Style Options: Customize icon size, color, message and optional text labels.
Icon Selection: Choose from three different icons from Lucide Icons to best match your design.

I hope you find this plugin useful! Your feedback is always welcome. Happy coding!

[Repository Copy Text](https://github.com/gpsblues/Publii-Plugin-Copy-text)

**Comment by bobmitro** [comment]
Great job! 
Thanks for contributing to the Publii community.
